app/routes/password/reset.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+const { Route, inject: { service } } = Ember;
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+
+  ajax: service(),
+
+  actions: {
+    resetPassword(password, passwordConfirmation) {
+      return this.get('ajax').request('/password/reset', {
+        method: 'POST',
+        data: {
+          password,
+          passwordConfirmation
```

Comment:
Do we need to send the confirmation, as well, or can we just use that on the client-side? I don't believe we have a concept on the server of a confirmation.